File: haikan-parent/haikan-manager-interface/src/main/java/com/haikan/manager/service/AdminService.java
package com.haikan.manager.service;

import java.util.List;

import com.haikan.pojo.TbAdmin;

import entity.PageResult;

/**
 * 管理员服务层接口
 * @author Administrator
 *
 */
public interface AdminService {

	/**
	 * 查询全部
	 * @return
	 */
	public List<TbAdmin> findAll();
	
	/**
	 * 分页查询
	 * @param pageNum
	 * @param pageSize
	 * @return
	 */
	public PageResult findPage(int pageNum,int pageSize);
	
	/**
	 * 增加管理员
	 * @param admin
	 */
	public void add(TbAdmin admin);
	
	/**
	 * 根据id查询管理员信息
	 * @param id
	 * @return
	 */
	public TbAdmin findOne(Long id);
	
	/**
	 * 修改管理员信息
	 * @param admin
	 */
	public void update(TbAdmin admin);
	
	/**
	 * 批量删除管理员
	 * @param ids
	 */
	public void delete(Long[] ids);
}

File: haikan-parent/haikan-manager-web/src/main/webapp/js/service/adminService.js
//管理员服务层
app.service('adminService',function($http){
	this.findAll=function(){
		return $http.get('../admin/findAll.do');
	}
	
	//分页查询
	this.findPage=function(pageNum,pageSize){
		return $http.get('../admin/findPage.do?pageNum='+pageNum+'&pageSize='+pageSize);
	}
	
	//增加管理员
	this.add=function(entity){
		return $http.post('../admin/add.do',entity);
	}
	
	//修改管理员
	this.update=function(entity){
		return $http.post('../admin/update.do',entity);
	}
	
	//根据id查询管理员信息
	this.findOne=function(id){
		return $http.get('../admin/findOne.do?id='+id);
	}
	
	//批量删除管理员信息
	this.dele=function(ids){
		return $http.get('../admin/delete.do?ids='+ids);
	}
	
});